Sources/API/Utils/OpenAPIClientAPI+Auth.swift

```diff
@@ -0,0 +1,5 @@
+import Auth
+
+extension OpenAPIClientAPI {
```

Comment:
This name is kinda odd 😄

Reply:
Agree, I will try to find another name 🙂

Reply:
I decided to do it later - because it is a default behaviour and we need to have one more overwritten template file. It requires to make a lot of changes in the integration as well, so the scope of the task will be significantly increased.